SocialNetwork/SocialNetwork.Core/Services/PostService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using SocialNetwork.Core.Interfaces;
+using SocialNetwork.Core.Models;
+namespace SocialNetwork.Core.Services
+{
+    public class PostService : IPostService
+    {
+        private readonly IRepository repository;
+        public PostService(IRepository repository)
+        {
+            this.repository = repository;
+        }
+        public IEnumerable<Post> GetPosts()
+        {
+            return repository.GetAll<Post>();
```

Comment:
In task about Post models it says: 'In order to sort posts for users we must provide this functionality'